Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi8/go-toolset:1.17.10 as builder
+FROM golang:1.18 as builder
```

Comment:
would it be better to use `registry.ci.openshift.org/openshift/release:golang-1.18 AS builder`

Reply:
Not sure of benefits here. Do you have a link to this image?